Description: Disables the @Generated annotation, since Java 11 this annotation
 is no longer in the standard JDK and causes build failures.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/istack-commons/maven-plugin/src/main/java/com/sun/istack/maven/ResourceGenMojo.java
+++ b/istack-commons/maven-plugin/src/main/java/com/sun/istack/maven/ResourceGenMojo.java
@@ -128,7 +128,7 @@
      * Mark generated sources with {@code @javax.annotation.Generated}.
      * @since 3.0.5
      */
-    @Parameter(property = "atGenerated", defaultValue = "true")
+    @Parameter(property = "atGenerated", defaultValue = "false")
     private boolean atGenerated;
 
     /**
